fluent-logger.gemspec

```diff
@@ -21,6 +21,8 @@ Gem::Specification.new do |gem|
   gem.require_paths = ['lib']
   gem.license       = "Apache-2.0"
 
+  gem.required_ruby_version = '>= 3.0'
```

Comment:
Seems `TCPSocket.new` supports `connect_timeout` and `resolv_timeout` since Ruby 3.0
https://github.com/ruby/ruby/blob/v3_0_0/ext/socket/tcpsocket.c#L23-L51